nakadi-producer-starter-spring-boot-2-test/pom.xml

```
@@ -26,12 +26,12 @@
         <dependency>
             <groupId>org.zalando.stups</groupId>
             <artifactId>tokens</artifactId>
-            <version>0.12.0-beta-2</version>
+            <version>0.14.0</version>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-jdbc</artifactId>
-            <version>2.0.3.RELEASE</version>
+            <version>2.5.5</version>
```

In the README you said it's tested with Spring-Boot 2.5.6 – I guess we should use that here too?
```suggestion
            <version>2.5.6</version>
```